sherpa/ui/utils.py

```diff
@@ -332,19 +333,85 @@ def reduce_ufunc(func):
 
 
 class ModelWrapper(NoNewAttributesAfterInit):
+    """Wrap up a model class so we can create instances easily.
+
+    The created instances can be used in a "model language" to create
+    complex expressions. If mdl1 and mdl2 have been created by
+    ModelWrapper then a user can say mdl1.n1 + mdl2.n2 to create model
+    instances named n1 and n2 and then return the expression which
+    represents their sum. Note that the model component is stored in
```

Comment:
I would understand "note the model components (plural) are stored" (meaning n1 and n2) or "that the model is stored" (singular, but not components). Can you clarify?

Reply:
Well,

    mdl1.m1 + mdl2.m2

is a composite model, but that is somewhat irrelevant here. What the line does is create two instances, "m1" from `mdl1` and "m2" from `mdl2`, and these are individually stored in the session object. Maybe this is all too esoteric for this docstring, but I wanted to indicate we can create a model instance within a "model expression", rather than making it look like you have to create components before combining them. Perhaps we need to re-organise this to say something like I(needs to be work-shopped):

    Creates a model instance with a given name - you can say mdl.mname or mdl("mname") - and this stores the
    instance in the session object with the name "mname", over-writing any previous component with this name.
    If mdl1 and mdl2 have been created by ModelWrapper then a user can say mdl1.n1 + mdl2.n2 to create model
    instances named n1 and n2 and then return the expression which represents their sum.

Reply:
Creates a model instance with a given name - you can say `mdl.mname` or `mdl("mname")` - and either syntax stores the model
instance in the session object with the name "mname", over-writing any previous component with this name.
If mdl1 and mdl2 have been created by ModelWrapper then a user can say `mdl1.n1 + mdl2.n2` to create model
instances named n1 and n2 and then return the expression which represents their sum.

Reply:
Just some slight modification on your suggestion which I think is clearer than the original suggestion.